crates/sdk/src/network/mod.rs

```diff
@@ -0,0 +1,86 @@
+use serde::{Deserialize, Serialize};
+
+pub mod prover;
+
+#[derive(Debug, Default, Clone)]
+pub struct NetworkClientCfg {
+    pub endpoint: Option<String>,
+    pub ca_cert_path: Option<String>,
+    pub cert_path: Option<String>,
+    pub key_path: Option<String>,
+    pub domain_name: Option<String>,
+    pub proof_network_privkey: Option<String>,
+}
+
+#[derive(Debug, Default, Deserialize, Serialize, Clone)]
+pub struct ProverInput {
+    pub elf: Vec<u8>,
+    pub public_inputstream: Vec<u8>,
+    pub private_inputstream: Vec<u8>,
+    pub shard_size: u32,
+    // pub execute_only: bool,
+    pub composite_proof: bool,
+    pub receipt_inputs: Vec<Vec<u8>>,
+    pub receipts: Vec<Vec<u8>>,
+    pub asset_url: String,
+    pub proof_results_path: String,
+}
+
+#[derive(Debug, Default, Deserialize, Serialize, Clone)]
+pub struct ProverResult {
+    pub total_steps: u64,
+    pub split_cost: u64,
+    pub output_stream: Vec<u8>,
+    pub proof_with_public_inputs: Vec<u8>,
+    pub stark_proof: Vec<u8>,
+    pub public_values: Vec<u8>,
+    pub receipt: Vec<u8>,
+    pub elf_id: Vec<u8>,
+}
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::{utils, ProverClient};
+    use std::{env, fs};
+    use zkm_core_machine::io::ZKMStdin;
+    use zkm_prover::ZKMVerifyingKey;
+    #[ignore]
+    #[test]
+    fn test_proof_network_fib() {
+        utils::setup_logger();
+        let endpoint =
```

Comment:
Can we create an default function for NetworkClientCfg?  in default, we try to load the env variables.